mysql-test/suite/rocksdb/r/mysqldump.result

```diff
@@ -76,7 +76,7 @@ UNLOCK TABLES;
 
 rollback;
 connection con1;
-1
+0
```

Comment:
Unsure, that'ts a good catch. That looks to be the result of the grep for STWCS which should remain the same due to the --single-transaction flag. I tested that specifically when applying part of one of the patches but must have munged it somehow when applying or fixing another of the patches.